desktop/release-notes.md

```diff
@@ -24,6 +24,75 @@ Take a look at the [Docker Public Roadmap](https://github.com/docker/roadmap/pro
 
 For frequently asked questions about Docker Desktop releases, see [FAQs](faqs/general.md/#releases)
 
+## Docker Desktop 4.12.0
+2022-09-01
+
+> Download Docker Desktop
+>
+> {% include desktop-install.html %}
+
+## New
+
+- Minimum version to install or update Docker Desktop on macOS is now 10.15
+- Improved overall performance issues caused by calculating disk size. Related to [docker/for-win#9401](https://github.com/docker/for-win/issues/9401).
+- Added the experimental docker daemon with the containerd image store.
+- Allow Docker Desktop to run untagged image (https://github.com/docker/for-mac/issues/6425)
+- Add search capabilities to Docker Extension Marketplace. Fixes [docker/roadmap#346](https://github.com/docker/roadmap/issues/346).
+- [mac] Quit Docker Desktop completely from the Dock menu (https://github.com/docker/for-mac/issues/6440)
+- Enable ComposeV2 after factory reset
+- Copied terminal contents do not contain non-breaking spaces anymore
+- Upgrade kernel to 5.10.124
+- Enables ComposeV2 by default on new installations/if there is no previous configuration.
+- Prevents users on ARM macs without Rosetta installed from switching back to ComposeV1 (only has intel binaries).
+- Added View menu with Zoom In, Zoom Out, and Actual Size options to control the size of the Docker Desktop UI.
+- Descending default sort order for volumes size and created column, containers started column
+- Defined sort order for container/compose state column running > some running > paused > some paused > exited > some exited > created
+- Re-organized container row actions
+- Added compose stop button if any related container is stoppable.
+- Individual compose containers are now deletable from overview
+- Runs every command on the Quickstart guide immediately.
+- Remove workaround for virtiofsd <-> qemu protocol mismatch on Fedora 35, as it is no longer needed. Fedora 35 users should upgrade the qemu package to the most recent version (qemu-6.1.0-15.fc35 as of the time of writing).
+- Implemented an integrated terminal for containers.
+- Updated the tooltip design.
+- Show address for external links.
```

Comment:
I have no idea what this means. 😅

Reply:
Taken from the [PR](https://github.com/docker/pinata/pull/18547), the description there sounds like a better candidate for our release notes: "Adds a tooltip that displays the link address for all external links by default". What do you think?